src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -25,7 +25,13 @@ class CBSpriteNode: SKSpriteNode {
 
     // MARK: - Properties
     @objc var spriteObject: SpriteObject
-    @objc var currentLook: Look?
+    @objc var currentLook: Look? {
+        didSet {
+            guard let stage = self.scene as? StageProtocol else { return }
+            if !self.spriteObject.isBackground() { return }
```

Comment:
Please also add a test case documenting that `notifyBackgroundChange` is not called for normal objects which are not background objects.

Reply:
Thanks for the suggestion, I added a new test.